src/main/java/be/seeseemelk/mockbukkit/inventory/InventoryMock.java

```diff
@@ -251,161 +253,214 @@ public InventoryType getType()
 	@Override
 	public int getMaxStackSize()
 	{
-		// TODO Auto-generated method stub
-		throw new UnimplementedOperationException();
+		return 64;
 	}
 	
 	@Override
-	public void setMaxStackSize(int size)
-	{
-		// TODO Auto-generated method stub
-		throw new UnimplementedOperationException();
-		
-	}
+	public void setMaxStackSize(int size) {}
 	
 	@Override
 	public HashMap<Integer, ItemStack> removeItem(ItemStack... items) throws IllegalArgumentException
 	{
-		// TODO Auto-generated method stub
-		throw new UnimplementedOperationException();
+		for (int i = 0; i < this.items.length; i++) {
+			if(items[i].equals( this.items[i] )){
+				items[i] = null;
+			}
+		}
+		return new HashMap<>();
+
```

Comment:
This suspicious return statement gives me impression, that this doesn't work as it should.

Reply:
well, it worked on my projects on PatzMC

Reply:
but i'll fix that return statement